play-services-core/src/main/res/layout/activity_location_sharing_settings.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:title="@string/service_name_google_location_sharing"
+        android:layout_height="?attr/actionBarSize"/>
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/avatar_iv"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        app:layout_constraintTop_toBottomOf="@id/toolbar"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_marginTop="32dp"
+        android:layout_marginStart="24dp"/>
+
+    <TextView
+        android:id="@+id/account_email_tv"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textSize="18sp"
+        android:textColor="@color/gm3_ref_palette_neutral20"
```

Comment:
Directly using colors breaks in dark mode, please use theme attributes instead.
<img width="480" height="960" alt="image" src="https://github.com/user-attachments/assets/0aab7528-99d8-4678-b0ba-e7ba27dd4b12" />